trieste/models/model_interfaces.py

```diff
@@ -678,6 +696,7 @@ def perfom_optimization_step() -> None:  # alternate with natgrad optimizations
     GPR: GaussianProcessRegression,
     SGPR: GaussianProcessRegression,
     VGP: VariationalGaussianProcess,
+    SVGP: SparseVariational
```

Comment:
you mentioned that you would rename `SparseVariational` into something else for consistency?

Reply:
I did originally but that caused some tests to fail so I changed it back (commit e2003f9). What would be the best way to handle this sort of thing in future?

Reply:
feel free to suggest a better naming, you will just need to adjust the tests as well wherever they use the old name
dont hesitate to change the tests, especially for these kind of non-substantive changes

Reply:
ok sure, I think for the moment I'm happy with SparseVariational, as Henry pointed out SparseVariationalGaussianProcess gets quite long